admin-dev/themes/new-theme/js/pages/import/FormFieldToggle.js

```diff
@@ -128,8 +128,10 @@ export default class FormFieldToggle {
    * @param {int} entity
    */
   loadAvailableFields(entity) {
+    const url = -1 === window.location.href.indexOf('index.php') ? '../../../ajax.php' : '../../../../ajax.php';
```

Comment:
im really not happy with this fix. :(

Reply:
me neither but when we will migrate the Import 2nd step, this will be updated with the real url that come from the router: am I right?

Reply:
@PierreRambaud for the history, this url still come from legacy, and will be migrated in 1.7.6 imo.

Reply:
I said nothing :(

Reply:
> when we will migrate the Import 2nd step, this will be updated with the real url that come from the router: am I right?

yes, it should be removed after all import controller is migrated.